package.json

```diff
@@ -75,7 +77,7 @@
     "eslint": "^7.16.0",
```

Comment:
Not a functional change, but I generally like to see `dependencies` and `devDependencies` closer together